src/Module/Home.php

```diff
@@ -69,7 +69,7 @@ protected function content(array $request = []): string
 			$customHome = $homeFilePath;
 
 			if (file_exists($cssFilePath)) {
-				DI::page()['htmlhead'] .= '<link rel="stylesheet" type="text/css" href="' . DI::baseUrl()->get() . '/home.css' . '" media="all" />';
+				DI::page()['htmlhead'] .= '<link rel="stylesheet" type="text/css" href="' . DI::baseUrl() . '/home.css' . '" media="all" />';
```

Comment:
Please use `Page->registerStylesheet` instead.
```suggestion
				DI::page()->registerStylesheet('home.css', 'all');
```